### ISSUE TITLE
Any plan to release checkpoints after pre-training?

### ISSUE THREAD

**Issue by xunannancy** [NVlabs/VILA#2]
I really like the insights provided in VILA paper. I am wondering, if there is any plan to release model checkpoints after pre-training only? Many thanks!

**Comment by Efficient-Large-Language-Model** [comment]
Yes, we will release them soon.

**Comment by hkunzhe** [comment]
Same request. Looking forward to it !

**Comment by Lyken17** [comment]
we have released the intermediate the checkpoints https://huggingface.co/collections/Efficient-Large-Model/vila-15-checkpoints-intermediate-6674dd0c20ff491d38f8299d

Free feel to try it out and share your feedbacks.

**Comment by Lyken17** [comment]
@caoshiyi is working on LoRA support for VILA and can share some insights.

**Comment by hkunzhe** [comment]
@Lyken17 Hi, It seems that the above link is empty.

**Comment by Lyken17** [comment]
yeah, we are still working on that and please stay tuned. Will release the middle checkpoints in the middle july.

**Comment by ROIM1998** [comment]
@Lyken17 Hello and thanks for your great work. 
I found that instruction-finetuned VILA's weights are released under the cc-by-nc-4.0 license and I guess probably the reason is that the sharegpt4v dataset is used for training. I wonder if it's possible that the intermediate/pretrained weights would be released under a more permissive license that allows commercial usage since sharegpt4v is not used during pretraining. Thanks!

**Comment by Lyken17** [comment]
Hi Bowen @ROIM1998 

I am afraid we have to stick with cc-nc-4.0 license for all VILA weights due regulation terms. However, we have made all our training receipts and codebase public so you can train your own one.
